### PR TITLE
Add `--show-individual-scores` CLI flag

### PR DESCRIPTION
**Pull request by shihchengli**
## Description
Add `--show-individual-scores` so all scores for individual targets will be shown.

## Example / Current workflow
Run this CLI
```
chemprop train \
-i data/regression/mol/mol.csv \
--epochs 1 \
--task-type regression \
--ensemble-size 2 \
--show-individual-scores
```

You will see individual scores for each target after each model training is finished.
```
INFO:__main__:Entire Test Set results:
INFO:__main__:entire_test/mse: 1.035393409031115
INFO:__main__:Entire Test Set individual results:
INFO:__main__:entire_test/lipo/mse: 1.035393409031115
```

## Relevant issues
#719

## Checklist
- [ ] linted with flake8?
- [ ] (if appropriate) unit tests added?

### CODE REVIEW

**Review by hwpang — COMMENTED** [review]
Can we add a CLI test? It doesn't have to be a new one, you can just add this argument to an existing CLI test

**Review by hwpang — APPROVED** [review]
LGTM

File: chemprop/cli/train.py [hunk 1]
```diff
@@ -910,6 +893,44 @@ def train_model(
                 args.weight_column,
                 args.no_header_row,
             )
+            input_cols = (args.smiles_columns or []) + (args.reaction_columns or [])
+            target_cols = columns[1:] if len(input_cols) == 0 else columns[len(input_cols) :]
+
+            individual_scores = dict()
+            for metric in model.metrics[:-1]:
```
Comment (KnathanM) [inline chemprop/cli/train.py:900]: Why do you do `model.metrics[:-1]` instead of `model.metrics`?

Comment (hwpang) [reply chemprop/cli/train.py:900]: The last item in metrics is not a metric but a criterion, or the loss function

Comment (shihchengli) [reply chemprop/cli/train.py:900]: The last one is the criterion for model training. I think we don't want to report this as a metric on the test set?
